setup.cfg

```diff
@@ -42,7 +42,7 @@ envlist=
     default
     tensorflow
     torch
-;    prophet
+    prophet
 ;    all
```

Comment:
What is the deal with `all` being commented out?

Reply:
I'm not sure to be honest, its been uncommented now anyway